project.clj

```diff
@@ -53,10 +62,12 @@
                   ["vcs" "commit"]
                   ["vcs" "push"]]
 
-  :eastwood {:add-linters [:unused-fn-args :unused-locals :unused-namespaces
+  :aliases {"smoke" ["do" ["modules" "puff"] ["clean"] ["check"] ["kibit"] ["cljfmt" "check"]]}
+
+  :eastwood {:add-linters [:unused-fn-args
+                           :unused-locals
+                           :unused-namespaces
                            :unused-private-vars]
              :namespaces  [:source-paths]}
 
-  :global-vars {*warn-on-reflection* true}
```

Comment:
Does one of the linters handle this or are you not interested in doing it any longer?

Reply:
I turned it on, and had heaps of warnings from 3rd-party libraries. 

Turned it off again and use 'lein check' specifically to target our own source. Not entirely sure I have it covered properly, need to take a look.